### ISSUE TITLE
about the softmax in AGNNConv class

### ISSUE THREAD

**Issue by pengfei** [dmlc/dgl#1099]
Do I need to use the node features of the whole graph as parameters or do I need to use all the neighbor node features of a desired node as the 'feat ' parameter?
In the original paper, a softmax operation was performed on the neighbors of a node.
If all node features are used as parameters, softmax operation is performed on all nodes, it may be a bug.

**Comment by yzh119** [comment]
You need the node features of the whole graph.
AGNNConv uses DGL's [edge_softmax](https://docs.dgl.ai/api/python/nn.pytorch.html?highlight=edge_softmax#dgl.nn.pytorch.softmax.edge_softmax), which is performed on the in-coming edges of each node.

**Comment by yzh119** [comment]
Such softmax-on-incoming-edges operation is common in GNN's such as GAT and Transformers, so we create a module to do this for both PyTorch and MXNet (and Tensorflow in the near future).

**Comment by pengfei** [comment]
Thank you for your reply
